cg/meta/clean/demultiplexed_flow_cells.py

```diff
@@ -257,6 +257,12 @@ def archive_sample_sheet(self):
         globbed_unaligned_paths: Path.glob = Path(self.path).glob(
             DemultiplexingDirsAndFiles.UNALIGNED_DIR_NAME + ASTERISK
         )
+        if len(list(globbed_unaligned_paths)) == 0:
```

Comment:
```suggestion
        if not list(globbed_unaligned_paths):
```
I think empty lists in python are False